### PR TITLE
working example

### PR DESCRIPTION
**Pull request by kreier**
Just 7 additional lines in the basic.forever() function and a fix to the setMaxiumumAccelerationFlagIfExceededMaxium() function

### CODE REVIEW

File: eggs.js [hunk 1]
```diff
@@ -200,17 +190,11 @@ function showNegativeFeedback() {
...
-function setMaximumAccelerationFlagIfExceededMaximum(accelerationBeyondMaximum: boolean) {
-
-    if (maxAccelerationWasExceeded) {
-
-
-
-        let accelerationBeyondMaximum = true
-
-
-
```
Comment (matthiaskreier) [inline eggs.js:213]: Where is this function now?

File: eggs.js [hunk 2]
```diff
@@ -18,34 +18,24 @@ basic.forever(function () {
...
 
-
     //create an array of the three acceleration values and the magnitude
     let accelerationAndMagnitude = [accelerationValues[0],accelerationValues[1],accelerationValues[2],accelerationMagnitude]
 
-
     //check if the acceleration magnitude is more than is allowed by the system
     let accelerationBeyondMaximum = isAccelerationMoreThanIsAllowed(accelerationMagnitude)
 
-
```
Comment (matthiaskreier) [inline eggs.js:34]: Seems really condensed now

File: docs/shockdetector_with_reset.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    //The reason I have included it here is to give an example of separating the processing of inputs and variables from
+    //responses to those inputs. This separation of responsibilities allows for testing to make sure that each works correctly
+    //in isolation. This can be really helpful in debugging.
+    if (exceededMaximum == true) {
+        showNegativeFeedback()
+    }
+    else {
+        showPositiveFeedback()
+    }
+}
```
Comment (matthiaskreier) [inline docs/shockdetector_with_reset.js:106]: That's a lot of new code!

Comment (matthiaskreier) [inline docs/shockdetector_with_reset.js:106]: And one more new function.

File: eggs.js [hunk 4]
```diff
@@ -200,17 +190,11 @@ function showNegativeFeedback() {
...
+    if (!maxAccelerationWasExceeded) {
+        maxAccelerationWasExceeded = true
 
-function setMaximumAccelerationFlagIfExceededMaximum(accelerationBeyondMaximum: boolean) {
-
-    if (maxAccelerationWasExceeded) {
-
-
-
-        let accelerationBeyondMaximum = true
```
Comment (matthiaskreier) [inline eggs.js:210]: The only functionality is changing a global variable? Why then let? It's not exported!

**Review by matthiaskreier — APPROVED** [review]
The changes seem sufficient.

**Review by kreier — COMMENTED** [review]
Few can approve, but the code works.

**Pull request merged**